content/W2015/W15-blog-caleybrock.md

```diff
@@ -1,7 +1,7 @@
 Title: Get to Know: Caley Brock
 Date: 2015-01-13 19:00
 Category: Blog
-Tags: Get to Know
+Tags: get to know
 Slug: get-to-know-caley-brock
 Author: Julia Nguyen
 Summary: This week I sat down with Caley Brock, a fourth year student to talk about her transition into CS from arts, her passion for UI/UX, and how her support network keeps her grounded.
```

Comment:
Re: the 80 character line breaks though, just wanted to give you a heads up that due to the metadata parsing, this line can't be split up :)